oeso-gric.owl

```diff
@@ -0,0 +1,2564 @@
+
+    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#IrrigationSystem">
+        <rdfs:subClassOf rdf:resource="http://www.opensilex.org/vocabulary/oeso#Actuator"/>
+        <rdfs:label xml:lang="en">irrigation system</rdfs:label>
+        <rdfs:label xml:lang="fr">système d&apos;irrigation</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.opensilex.org/vocabulary/oeso#Leaf -->
+
+    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Leaf">
+        <rdfs:subClassOf rdf:resource="http://www.opensilex.org/vocabulary/oeso#ScientificObject"/>
+        <dc:source>skos:closeMatch http://www.ontobee.org/ontology/rdf/PO?iri=http://purl.obolibrary.org/obo/PO_0000036</dc:source>
+        <rdfs:comment xml:lang="en">A phyllome vascular system (PO:0025206) that includes the totality of the portions of  vascular tissue in their specific arrangement in a vascular leaf  (PO:0009025). [database_cross_reference:  POC:Ramona_Walls]</rdfs:comment>
+        <rdfs:label xml:lang="fr">feuille</rdfs:label>
+        <rdfs:label xml:lang="en">leaf</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.opensilex.org/vocabulary/oeso#Lens -->
+
+    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#Lens">
+        <rdfs:subClassOf rdf:resource="http://www.opensilex.org/vocabulary/oeso#SensingDevice"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.opensilex.org/vocabulary/oeso#hasBrand"/>
+                <owl:maxCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:maxCardinality>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.opensilex.org/vocabulary/oeso#aperture"/>
+                <owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:cardinality>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.opensilex.org/vocabulary/oeso#focalLength"/>
+                <owl:cardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:cardinality>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:label xml:lang="en">lens</rdfs:label>
+        <rdfs:label xml:lang="fr">objectif</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.opensilex.org/vocabulary/oeso#LevelMeasurementRainGauge -->
+
+    <owl:Class rdf:about="http://www.opensilex.org/vocabulary/oeso#LevelMeasurementRainGauge">
+        <rdfs:subClassOf rdf:resource="http://www.opensilex.org/vocabulary/oeso#PrecipitationSensor"/>
+        <dc:source>skos:closeMatch Level measurement rain gauge (LRG) [IOM-99_FI-RI 2.2.1e] http://www.wmo.int/pages/prog/www/IMOP/publications/IOM-99_FI-RI.pdf</dc:sour
```

Comment:
Are there 2 labels in english for grain ? grain and kernel ?

Reply:
yes, for wheat either grain or kernel can be used

Reply:
we distinguish the common term "seed" (PO:0025496) and the more specific term "grain" (PO:0030104) used for Poacea and synonym to caryopsis fruit (and kernel)

Reply:
Maybe you should create 2 concepts or keep the less specific

Reply:
I'll remove the "kernel" and "caryopisi fruit" as pseudo synonyms and replace the "grain" label by "caryopsis fruit" in order to keep the PO term